components/automate-chef-io/content/docs/backup.md

```diff
@@ -293,41 +305,5 @@ y
 Success: Backups deleted
 ```
 
-## Troubleshooting
-
-### Update the FQDN
-
-Chef Automate's restore process expects to run on a host with the same FQDN as the host FQDN at the time of the backup.
-If the restore runs on a host with a different FQDN, you will need to patch the FQDN and TLS certificate at the end of the restore process.
-
-1. On the restored host, generate a configuration file on the host:
-
-    ```shell
-    chef-automate init-config
-    ```
-
-2. Create a `patch.toml` file that contains the FQDN, TLS certificate, and TLS key entries from the generated configuration:
-
-   ```toml
-    [global.v1]
-    fqdn = "<new-fqdn>"
-
-    [[global.v1.frontend_tls]]
-    # The TLS certificate for the load balancer frontend.
-    cert = """-----BEGIN CERTIFICATE-----
-    <certificate-for-new-fqdn>
-    -----END CERTIFICATE-----
-    """
-
-    # The TLS RSA key for the load balancer frontend.
-    key = """-----BEGIN RSA PRIVATE KEY-----
-    <key-for-new-fqdn>
-    -----END RSA PRIVATE KEY-----
-    """
-   ```
-
-3. Patch the restored Chef Automate installation with the new FQDN, TLS certificate, and TLS key configuration entries:
-
-   ```shell
-   chef-automate config patch </path/to/patch.toml>
-   ```
+## References
```

Comment:
Added since I'm not sure people are finding the CLI docs on their own. (Not sure it will make a difference, but it makes me feel better.)

Reply:
Good enough reason for me.